src/modules/popmotion-controller/popmotion-controller.ts

```diff
@@ -92,82 +115,89 @@ export const popmotionController = (config: PopmotionControllerConfig = {}): Run
-      const atlasPointsCache: any[] = [];
-      let prevDiff = -1;
-      function removeFromEventCache(e: PointerEvent) {
-        // Remove this event from the target's cache
-        for (let i = 0; i < eventCache.length; i++) {
-          if (eventCache[i].pointerId == e.pointerId) {
-            eventCache.splice(i, 1);
-            atlasPointsCache.splice(i, 1);
-            break;
-          }
-        }
-      }
-
-      function pointerDown(e: PointerEvent) {
-        eventCache.push(e);
-        atlasPointsCache.push({ ...((e as any).atlas || {}) });
-      }
-      function pointerMove(e: PointerEvent) {
-        for (let i = 0; i < eventCache.length; i++) {
-          if (e.pointerId == eventCache[i].pointerId) {
-            eventCache[i] = e;
-            atlasPointsCache[i] = { ...((e as any).atlas || {}) };
-            break;
-          }
-        }
-        if (eventCache.length == 2) {
-          const curDiff = Math.abs(eventCache[0].clientX - eventCache[1].clientX);
-
-          // - - 2 - - 6- - - - 10
-          const xDiff =
-            atlasPointsCache[0].x > atlasPointsCache[1].x
-              ? atlasPointsCache[0].x - atlasPointsCache[1].x
-              : atlasPointsCache[1].x - atlasPointsCache[0].x;
-          const yDiff =
-            atlasPointsCache[0].y > atlasPointsCache[1].y
-              ? atlasPointsCache[0].y - atlasPointsCache[1].y
-              : atlasPointsCache[1].y - atlasPointsCache[0].y;
-
-          if (prevDiff > 0) {
-            if (curDiff > prevDiff) {
-              runtime.world.zoomTo(
-                // Generating a zoom from the wheel delta
-                0.95,
-                { x: xDiff / 2, y: yDiff / 2 },
-                true
-              );
-            }
-            if (curDiff < prevDiff) {
-              runtime.world.zoomTo(
-                // Generating a zoom from the wheel delta
-                1.05,
-                { x: xDiff / 2, y: yDiff / 2 },
-                true
-              );
-            }
-          }
-
-          // Cache the distance for the next move event
-          prevDiff = curDiff;
-        }
-      }
-
-      function pointerUp(e: PointerEvent) {
-        // Remove this pointer from the cache and reset the target's
-        // background and border
-        removeFromEventCache(e);
-        // If the number of pointers down is less than two then reset diff tracker
-        if (eventCache.length < 2) {
-          prevDiff = -1;
-        }
+      // const eventCache: PointerEvent[] = [];
+      // const atlasPointsCache: any[] = [];
+      // let prevDiff = -1;
+      // function removeFromEventCache(e: PointerEvent) {
+      //   // Remove this event from the target's cache
+      //   for (let i = 0; i < eventCache.length; i++) {
+      //     if (eventCache[i].pointerId == e.pointerId) {
+      //       eventCache.splice(i, 1);
+      //       atlasPointsCache.splice(i, 1);
+      //       break;
+      //     }
+      //   }
```

Comment:
is it ok to remove the pointer event logic which is not used?

Reply:
Yeah, that's totally fine! This file has been slowly getting smaller and more defined. It's all in the git history